.github/workflows/master.yml

```diff
@@ -117,7 +117,7 @@ jobs:
           - python-version: 3.8
```

Comment:
@itholic can we change ` pyarrow-version: 0.15.1` above to ` pyarrow-version: 1.0.1`? I think we should also test PyArrow 1.x as well.

Reply:
Sounds good! Thanks :)